Header header
string name
string ip
int32 port
int32 status